lib/src/util/hash_scripts.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:convert';
+
+import 'package:crypto/crypto.dart';
+import 'package:html/dom.dart';
+import 'package:html/parser.dart' show parse;
+import 'package:sidekick_core/sidekick_core.dart';
+
+void hashScripts({required bool shouldHash, required Hash hash}) {
```

Comment:
Shouldn't this not just not be called if we do not want to hash? So we may just get rid of this bool?  🤔

Reply:
Yes we could do it this way too,
I just thought this will be the non-removing part to easily find this method later again if you want to use it or if you have environments to configure it